### PR TITLE
adding kobiton sample project

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: integrations/kobiton_integration_demo_project/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# kobiton integration with codemagic
```
Comment (nevercode-kevin) [inline integrations/kobiton_integration_demo_project/README.md:1]:
```suggestion
# Kobiton integration with Codemagic
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: integrations/kobiton_integration_demo_project/codemagic.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+              script: |
+                keychain add-certificates
+            - name: Increment build number
+              script: |
+                #!/bin/sh
+                set -e
+                set -x
+                cd $FCI_BUILD_DIR/ios
+                # agvtool new-version -all $(($BUILD_NUMBER + 1))
+                agvtool new-version -all $(($(app-store-connect get-latest-testflight-build-number "$APP_STORE_APP_ID") + 1))
```
Comment (nevercode-kevin) [inline integrations/kobiton_integration_demo_project/codemagic.yaml:62]:
```suggestion
            - name: Increment build number
              script: |
                cd $FCI_BUILD_DIR/ios
                # agvtool new-version -all $(($BUILD_NUMBER + 1))
                agvtool new-version -all $(($(app-store-connect get-latest-testflight-build-number "$APP_STORE_APP_ID") + 1))
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: integrations/kobiton_integration_demo_project/codemagic.yaml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            - name: Set up code signing settings on Xcode project
+              script: |
+                xcode-project use-profiles --warn-only
+            - name: Build ipa for distribution
+              script: |
+                xcode-project build-ipa --workspace "$FCI_BUILD_DIR/ios/$XCODE_WORKSPACE" --scheme "$XCODE_SCHEME"
```
Comment (nevercode-kevin) [inline integrations/kobiton_integration_demo_project/codemagic.yaml:68]:
```suggestion
                xcode-project build-ipa --workspace "$CM_BUILD_DIR/ios/$XCODE_WORKSPACE" --scheme "$XCODE_SCHEME" 
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by icarusdust** [comment]
Changes made!

File: integrations/kobiton_integration_demo_project/codemagic.yaml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+              script: |
+                # For information about Codemagic CLI commands visit: https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/README.md
+                # For details about the --type paramater below - https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/fetch-signing-files.md#--typeios_app_adhoc--ios_app_development--ios_app_inhouse--ios_app_store--mac_app_development--mac_app_direct--mac_app_store--mac_catalyst_app_development--mac_catalyst_app_direct--mac_catalyst_app_store--tvos_app_adhoc--tvos_app_development--tvos_app_inhouse--tvos_app_store
+                app-store-connect fetch-signing-files "$BUNDLE_ID" --type IOS_APP_STORE --create
+            - name: Use system default keychain
+              script: |
+                keychain add-certificates
+            - name: Increment build number
+              script: |
+                cd $FCI_BUILD_DIR/ios
```
Comment (nevercode-kevin) [inline integrations/kobiton_integration_demo_project/codemagic.yaml:57]:
```suggestion
                cd $CM_BUILD_DIR/ios
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: integrations/kobiton_integration_demo_project/codemagic.yaml [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                  # https://appstoreconnect.apple.com/access/api
+                - certificate_credentials # <-- (Includes CERTIFICATE_PRIVATE_KEY - Put your Certificate Private Key here)
+                - other
+            vars:
+                XCODE_WORKSPACE: "YOUR_WORKSPACE_NAME.xcworkspace" # <-- Put the name of your Xcode workspace here
+                XCODE_SCHEME: "YOUR_SCHEME_NAME" # <-- Put the name of your Xcode scheme here
+                BUNDLE_ID: "YOUR_BUNDLE_ID_HERE" # <-- Put your Bundle Id here e.g com.domain.myapp
+                APP_STORE_APP_ID: 1555555551 # <-- Put the app id number here. This is found in App Store Connect > App > General > App Information
+            node: v16.11.1
+            xcode: 12.5
```
Comment (nevercode-kevin) [inline integrations/kobiton_integration_demo_project/codemagic.yaml:26]:
```suggestion
            xcode: 13.3.1
```

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**